test/system/090-events.bats

```diff
@@ -291,3 +291,22 @@ EOF
     _events_container_create_inspect_data journald
     _events_container_create_inspect_data file
 }
+
+@test "event with too big labels" {
+    # test that events entries are not truncated when reading from syslog
+
+    skip_if_remote "remote does not support --events-backend"
+
+    cname=test-$(random_string 30 | tr A-Z a-z)
+
+    local label_file="$BATS_TEST_TMPDIR/labels.txt"
```

Comment:
No such thing. You want `PODMAN_TMPDIR` https://github.com/containers/podman/blob/6e2e9ab2273f0ec6d2a993b8968b34b045509e02/test/system/helpers.bash#L81-L86

Reply:
You sure? Fore me it returns more than just `/tmp/`.
e.g 
`/tmp/bats-run-m6qynu/test/1/`

Reply:
it should be specific for each test case

Reply:
Oh, nice. TIL about `$BATS_TEST_TMPDIR`. But it's new in bats 1.4, and RHEL still uses 1.0 or 1.1, so yeah, sorry, you need to use `$PODMAN_TMPDIR`. It is unique to each `@test`.

Reply:
fixed